modules/op-release-notes-1-9.adoc

```diff
@@ -373,3 +373,42 @@ spec:
 // Savita Ashture
 
+[id="release-notes-1-9-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.1
+
+With this update, {pipelines-title} General Availability (GA) 1.9.1 is available on {product-title} 4.11 and later.
+
+[id="fixed-issues-1-9-1_{context}"]	
+== Fixed issues	
+
+* Before this update, the `tkn pac repo list` command did not run on Microsoft Windows. With this update, this issue is resolved and now you can run the `tkn pac repo list` command on on Microsoft Windows.
+
+* Before this update, {pac} watcher did not receive all of the configuration change events. With this update, this issue is resolved. The {pac} watcher is updated and does not miss the configuration change events.
+
+* Before this update, TaskRuns or PipelineRuns pods created by {pac} could not access custom certificates exposed by the user in the cluster. With this update, this issue has been resolved. You can now access custom certificates in the cluster from TaskRuns or PipelineRuns pods.
+
+* Before this update, the core interceptor, `tekton-triggers-core-interceptors`, used in the `Trigger` resource did not function after the Pipelines operator was upgraded to version 1.9, on a cluster which is enabled with FIPS. With this update, this issue is resolved. The core interceptor, `tekton-triggers-core-interceptors`, has been updated to TLS version 1.2, (because OpenShift uses MInTLS 1.2 for all it's components), which ensures that the functionality of the core interceptor runs accurately.
+
+* Before this update, when using a pipeline run with an internal OpenShift image registry, the URL to the image had to be hardcoded in the pipeline run definition. For example:
++
+[source,yaml]
+----
+...
+  - name: IMAGE_NAME
+    value: 'image-registry.openshift-image-registry.svc:5000/<test_namespace>/<test_pipelinerun>'
+...
+----
++
+When using in the context of {pac}, such hardcoded values prevents pipeline run definitions from being used in different clusters and namespaces.
++
+With this update, you can generalize the pipeline run definitions by using the dynamic template variables, instead of hardcoding the values for namespaces and pipeline run names. For example:
++
+[source,yaml]
+----
+...
+  - name: IMAGE_NAME
+    value: 'image-registry.openshift-image-registry.svc:5000/{{ target_namespace }}/{{ $(context.pipelineRun.name) }}'
+...
+----
+
+* Before this update, {pac} used the same GitHub token to fetch a remote task available in the same host on the default GitHub branch. With this update, this issue is now resolved. {pac} uses the same GitHub token to fetch a remote task from any GitHub branch.
```

Comment:
I would add word `only` (on the default GitHub branch) to the first sentence so that second sentence makes sense.